src/content/en/updates/2018/12/class-fields.md

```diff
@@ -0,0 +1,214 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Several proposals expand the existing JavaScript class syntax with new functionality. This article explains the new public class fields syntax in V8 v7.2 and Chrome 72, as well as the upcoming private class fields syntax.
+
+{# wf_updated_on: 2018-12-13 #}
+{# wf_published_on: 2018-12-13 #}
+{# wf_tags: javascript,chrome72 #}
+{# wf_featured_image: /web/updates/images/generic/info.png #}
+{# wf_featured_snippet: Several proposals expand the existing JavaScript class syntax with new functionality. This article explains the new public class fields syntax in V8 v7.2 and Chrome 72, as well as the upcoming private class fields syntax. #}
+{# wf_blink_components: Blink>JavaScript>Language #}
+
+# Public and private class fields {: .page-title }
+
+{% include "web/_shared/contributors/mathiasbynens.html" %}
+
+Several proposals expand the existing JavaScript class syntax with new functionality. This article
+explains the new public class fields syntax in V8 v7.2 and Chrome 72, as well as the upcoming
+private class fields syntax.
+
+Here’s a code example that creates an instance of a class named `IncreasingCounter`:
+
+```js
+const counter = new IncreasingCounter();
+counter.value;
+// logs 'Getting the current value!'
+// → 0
+counter.increment();
+counter.value;
+// logs 'Getting the current value!'
+// → 1
+```
+
+Note that accessing the `value` executes some code (i.e., it logs a message) before returning the
```

Comment:
Did you want to use the `Note:` syntax for the blue box?

Reply:
Not in this case. It’s part of the general flow, and doesn’t need a separate callout.